openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -243,6 +251,37 @@ def objective_callback(self, x, icase):
         """
```

Comment:
Make this a raw string, since it contains control characters.  (r""" )

Reply:
Thanks, changes done.